igvc_navigation/launch/localization.launch

```diff
@@ -59,25 +58,25 @@
                X velocity, Y velocity, Z velocity,
                roll velocity, pitch velocity, yaw velocity,
                X acceleration, Y acceleration, Z acceleration -->
-      <rosparam param="odom0_config">[false,  false, false,
-                               false, false, false,
-                               false, false, false,
-                               false, false, false,
-                               false, false, false]</rosparam>
+      <rosparam param="odom0_config">[false, false, false,
+                                      false, false, false,
+                                      true, true, false,
+                                      false, false, false,
+                                      false, false, false]</rosparam>
 
       <!-- IMU -->
       <param name="imu0" value="/imu"/>
       <!-- fused parameters same as above -->
       <rosparam param="imu0_config">[false, false, false,
-                                 false, false, true,
-                                 false, false, false,
-                                 false,  false,  true,
-                                 false,  false,  true]</rosparam>
+                                     false, false, false,
+                                     false, false, false,
+                                     false,  false, true,
+                                     true,  false,  false]</rosparam>
```

Comment:
only x acceleration? is this because of non-holonomic constraints?

Reply:
yes